### ISSUE TITLE
Rotation is very laggy

### ISSUE THREAD

**Issue by szaimen** [scaleflex/filerobot-image-editor#306]
Dear Scaleflex,

thank you for this great image editor!

Unfortunately it looks like the rotation is very laggy. 

Could you please look into improving this? Thanks a lot!

https://user-images.githubusercontent.com/42591237/222499140-adf0bd88-d498-4952-bb69-f1c7a301fb58.mp4

**Comment by AhmeeedMostafa** [comment]
@szaimen Yes we notice that, in fact it is intended behavior currently for avoiding performance issue while implementation, but we are considering its improvement once we have the time for it in the future.

**Comment by szaimen** [comment]
Thank you! :)

**Comment by skjnldsv** [comment]
Any news @AhmeeedMostafa ?
